### PR TITLE
Add event date

### PR DESCRIPTION
**Pull request by myang2020**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddEventCommand.java [hunk 1]
```diff
@@ -18,7 +19,8 @@ public class AddEventCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates and adds an event to the address book. "
             + "Parameters: "
-            + PREFIX_EVENT_NAME + "EVENT_NAME";
+            + PREFIX_EVENT_NAME + "EVENT_NAME"
+            + PREFIX_DATE + "EVENT_DATE";
```
Comment (ChocoRagdoll) [inline src/main/java/seedu/address/logic/commands/AddEventCommand.java:23]: its better if there are exmaples, check other command classes for reference

File: src/main/java/seedu/address/storage/JsonAdaptedEvent.java [hunk 2]
```diff
@@ -23,15 +24,19 @@ class JsonAdaptedEvent {
 
     private final String eventName;
 
+    private final String eventDateTime;
```
Comment (ChocoRagdoll) [inline src/main/java/seedu/address/storage/JsonAdaptedEvent.java:27]: should this be eventDate or is eventDateTime okay

**Review by ChocoRagdoll — COMMENTED** [review]
lgtm, just some cosmetic issues

File: src/main/java/seedu/address/logic/commands/AddEventCommand.java (hunk 1 above)
Comment (myang2020) [reply src/main/java/seedu/address/logic/commands/AddEventCommand.java]: Added examples.

File: src/main/java/seedu/address/storage/JsonAdaptedEvent.java (hunk 2 above)
Comment (myang2020) [reply src/main/java/seedu/address/storage/JsonAdaptedEvent.java]: Fixed variable name to eventDate

File: src/test/java/seedu/address/storage/JsonSerializableEventBookTest.java [hunk 3]
```diff
@@ -34,16 +37,19 @@ public void toModelType_validEventDetails_success() throws IllegalValueException
...
     }
 
     @Test
     public void toModelType_duplicateEvents_throwsIllegalValueException() {
         // Setup
         UniquePersonList a = new UniquePersonList();
         a.setPersons(TypicalPersons.getTypicalPersons());
-        JsonAdaptedEvent jsonEvent1 = new JsonAdaptedEvent(new Event(new EventName("Event"), a));
+        JsonAdaptedEvent jsonEvent1 = new JsonAdaptedEvent(new Event(new EventName("Event"),
+                new EventDate("05-07-2024"), a));
```
Comment (myang2020) [inline src/test/java/seedu/address/storage/JsonSerializableEventBookTest.java:52]: Make 05-07-2024 default event date

**Pull request merged**